content/docs/regular_events/drop-in_sessions.md

```diff
@@ -14,22 +14,39 @@ weight: 1
-At the sessions they can ask the group about specific technical problems, or seek general technical advice.
+Whether you have general questions on a topic, or are seeking specific technical guidance on, or just want to chat with friendly faces, our drop-in session is the perfect place for you!
 The sessions offer a friendly, welcoming, non-judgemental environment to receive advice or assistance without concern about being judged on technical knowledge, expertise or understanding.
 
 As our organisers and helpers know, many of us working with software in the research community have been trained in domains where computing has not traditionally been a key element.
 There can be lots to learn in terms of new jargon and technical skills and we all have to start somewhere.
 
-We can help with:
+A few examples of what we can help out with are:
 
-- General programming problems — including helping to identify and fix bugs.
-- Cloud computing questions — including how to get started on Azure.
-- Software sustainability — best practices for testing, managing and packaging your code.
-- Reproducible research — ensuring software is developed to support long-term reproducibility, including techniques for packaging and archiving code.
-- High Performance Computing (HPC) — support and advice on the HPC resources available at the Turing.
+- General programming problems, including:
+    - Identifying and fixing bugs
+    - Setting up Python packages and environments
+- Cloud computing questions, including:
+    - How to get started on Microsoft Azure
+    - How to deploy code to the cloud
+    - Cloud computing and storage costs
+    - Getting access to GPU VMs on Microsoft Azure
+- Research Data Science issues, including:
+    - Data collection and selection process
+    - Experimental design (methods, baselines, metrics, ablation studies)
+    - Moving from a general idea to a machine learning pipeline
+- Software sustainability
+    - Best practices for testing, managing and packaging your code
+- Reproducible research
+    - Ensuring software is developed to support long-term reproducibility
+    - Including techniques for packaging and archiving code and data
+- High Performance Computing (HPC) support:
+    - HPC services resources available at the Turing
+    - HPC performance (parallellism, libraries, compilers)
 
 ## Joining sessions
 
-Our weekly sessions run for one hour from 13:30-14:30 every Tuesday and are hosted on [Zoom](https://github.com/alan-turing-institute/research-engineering-group/wiki/The-REGistry#video-call-links).
-Everyone is welcome to drop-in at any time.
-There is no need to register in advance.
+Our sessions are hosted on [Zoom](https://turing-uk.zoom.us/j/606996571?pwd=haUeR9-Ypwoy_-fxjvz1ySo_eocgsw) and sometimes hybrid. We run several week, so please see our [schedule](https://github.com/alan-turing-institute/research-engineering-group/wiki/Drop%E2%80%90in-Sessions#schedule) on the REG wiki.
```

Comment:
~We run several week~ We run several a week

Reply:
thanks! updated